packages/portal-proto/src/components/ImageViewer/MultipleImageViewer.tsx

```diff
@@ -70,19 +67,13 @@ export const MultipleImageViewer = ({
     ) as FilterSet,
   );
 
-  const { data, isFetching } = useImageViewer({
+  const { data, isFetching } = useImageViewerQuery({
     cases_offset,
     searchValues,
     case_id,
     caseFilters: filters,
   });
 
-  useEffect(() => {
-    return () => {
-      dispatch(resetEdgesState());
-    };
-  }, [dispatch]);
-
```

Comment:
This code was added to ensure that when the user navigates away from the image viewer tab (unmounts) and later returns to it, the tab will not display more than 10 slides upon remounting IIRC.  Will it still support this?

Reply:
I'm not sure that would be an issue with how the rtk query caching works but I'll check.